docs/content/v2025.1/explore/ysql-language-features/advanced-features/parallel-query.md

```diff
@@ -2,8 +2,6 @@
 title: Parallel queries
 linkTitle: Parallel queries
 description: Parallel queries in YSQL
-tags:
-  feature: early-access
 menu:
```

Comment:
@sushantrmishra now that parallel queries is GA, do any flag defaults change, or does behavior stay the same.